front/src/modules/trainschedule/components/Timetable/TimetableToolbar.tsx

```diff
@@ -77,59 +80,73 @@ const TimetableToolbar = ({
 
   const [deleteTrainSchedules] = osrdEditoastApi.endpoints.deleteTrainSchedule.useMutation();
 
+  const { trainScheduleIds, pacedTrainIds } = selectedTimetableItemIds;
+  const timetableItemIds = [...trainScheduleIds, ...pacedTrainIds];
```

Comment:
I think we should keep the info that we are manipulating the selected ids here (can be another name)
```suggestion
  const selectedIds = [...trainScheduleIds, ...pacedTrainIds];
```

Reply:
Done